t/10-validation.t

```diff
@@ -292,4 +292,44 @@ subtest 'flagged as not deletable' => sub {
   ok (!$v->run(), 'run is not deletable');
 };
 
+subtest 'per product flag and iRODS locations' => sub {
+  plan tests => 10;
+
+  my $rfh = _create_test_runfolder_8747();
+  my $ref = {
+    id_run => 8747,
+    runfolder_path      => $rfh->{'runfolder_path'},
+    analysis_path       => $rfh->{'analysis_path'},
+    archive_path        => $rfh->{'archive_path'},
+    qc_schema           => $qc_schema,
+    npg_tracking_schema => $tracking_schema
+  };
+  mkdir join q[/], $rfh->{'runfolder_path'}, 'npg_do_not_delete';
```

Comment:
Not sure this was used. I was expecting to see a validation returning false because of the folder. But don't think it is there.

Reply:
This test was not changed in this pr

Reply:
Removed, thanks